src/views/dashboard/index.js

```diff
@@ -36,12 +49,18 @@ class Dashboard extends Component {
         </div>
       ))
     )
+    const filters = (Array.isArray(this.props.filters) && this.props.filters) || []
```

Comment:
I guess you could just ensure it's an array with props and default to empty array with `defaultProps`....